thorlcr/activities/firstn/thfirstnslave.cpp

```diff
@@ -48,7 +48,6 @@ class CFirstNSlaveBase : public CSlaveActivity
     }
     virtual void start() override
     {
-        ActivityTimer s(slaveTimerStats, timeActivities);
```

Comment:
Removed to avoid duplicating ActivityTimer (which is also created in CFirstNSlaveGlobal::start)